include/fmt/core.h

```diff
@@ -279,6 +290,14 @@ struct monostate {};
 
 namespace detail {
 
+constexpr bool is_constant_evaluated() FMT_DETECTED_NOEXCEPT {
+#if FMT_HAS_IS_CONSTANT_EVALUATED
```

Comment:
This macro is not needed, please check `__cpp_lib_is_constant_evaluated` instead.